### ISSUE TITLE
Unable to load GadFly module in Julia 0.4

### ISSUE THREAD

**Issue by galenlynch** [dcjones/Gadfly.jl#647]
I've been unable to use GadFly for the last week or so, due to an error while loading the module. When I try to use the most recent version of GadFly (0.3.13) I receive the following error:
```
julia> using GadFly
ERROR: LoadError: LoadError: UndefVarError: LAB not defined
 in include at ./boot.jl:254
 in include_from_node1 at ./loading.jl:196
 in require at ./loading.jl:145
 in include at ./boot.jl:254
 in include_from_node1 at ./loading.jl:196
 in require at ./loading.jl:145
while loading /Users/galenlynch/.julia/v0.4/GadFly/src/Color.jl, in expression starting on line 14
while loading /Users/galenlynch/.julia/v0.4/GadFly/src/GadFly.jl, in expression starting on line 4
```

I'm using Julia Version 0.4.0-dev+6450

**Comment by darwindarak** [comment]
This should be fixed in c83e17b.  Can you check if this error occurs if you use the later commits?

**Comment by galenlynch** [comment]
Yes, that fixes it, thanks!
